layersvt/basic.cpp

```diff
@@ -30,7 +30,7 @@
 typedef VkResult(VKAPI_PTR *PFN_vkLayerBasicEXT)(VkDevice device);
 static PFN_vkLayerBasicEXT pfn_layer_extension;
 
-VK_LAYER_EXPORT VKAPI_ATTR VkResult VKAPI_CALL vkLayerBasicEXT(VkDevice device) {
+VKAPI_ATTR VkResult VKAPI_CALL basic_LayerBasicEXT(VkDevice device) {
     printf("In vkLayerBasicEXT() call w/ device: %p\n", (void *)device);
```

Comment:
The rename is for consistency: all layer commands are prefixed by basic_ instead of vk.  I do not see how it is a extension command matters?